py3status/modules/mpris.py

```diff
@@ -117,14 +117,8 @@
 
 
 def _get_time_str(microseconds):
-    seconds = int(microseconds / 1000000)
-    time = str(timedelta(seconds=seconds))
-    if time[0] == "0":
-        time = time[2:]
-        if time[0] == "0":
-            time = time[1:]
-
-    return time
+    delta = timedelta(seconds=microseconds // 1_000_000)
+    return str(delta).lstrip("0").lstrip(":").lstrip("0")
```

Comment:
(Suggestion) `lstrip("0:")`?

Reply:
`datetime.timedelta` is definitely missing a good `__format__` support. Both my and your suggestions change the behaviour for timedeltas shorter than 1 minute:

```
microseconds:
    'original'
    'mine'
    'yours'

123:
    '0:00'
    ':00'
    ''

1234567:  
    '0:01'
    ':01'
    '1'

12345678:
    '0:12'
    ':12'
    '12'
```

I'll change it to the original behavior and then we'll wait until `timedelta` can do the job itself.

Reply:
I have changed it and repushed.